cumulusci/cli/service.py

```diff
@@ -142,6 +142,16 @@ def callback(*args, **kwargs):
                     abort=True,
                 )
 
+            prompt_to_default_service = f"A default service already exists for service type {service_type}. Would you like to set this service as the new default?"
+            default_service_exists = (
+                True
+                if runtime.keychain.get_default_service_name(service_type) is not None
+                else False
```

Comment:
Debatable, but i very mildly prefer this:

```suggestion
            default_service_exists = runtime.keychain.get_default_service_name(service_type) is not None
```

But that's totally debatable.